### PR TITLE
[Ruby] Concepts Reference - Gigasecond 

### PR DESCRIPTION
**Pull request by kotp**
Reference: #279

### CODE REVIEW

File: languages/ruby/reference/exercise-concepts/gigasecond.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+== Gigasecond
```
Comment (kotp) [inline languages/ruby/reference/exercise-concepts/gigasecond.md]:
```suggestion
## Gigasecond
```
I am not sure where this documentation will end up, so unsure if we should use level 1 header, or level 2.

**Comment by ErikSchierboom** [comment]
@kotp Are you still interested in working on this PR?

**Comment by kotp** [comment]
> @kotp Are you still interested in working on this PR?

Not so much at the moment, and enough time has passed, not sure that bringing it in would add value for the upcoming v3.  We can always pick it up later if it is.

**Comment by ErikSchierboom** [comment]
@kotp Agreed!

**Pull request closed without merging**